homework/Repository.js

```diff
@@ -8,25 +8,34 @@ class Repository {
     this.repository = repository;
   }
 
-  /**
-   * Render the repository info to the DOM.
-   * @param {HTMLElement} container The container element in which to render the repository.
-   */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.repository, null, 2));
+    const theads = ['Repository:', 'Description:', 'Forks:', 'Updated:'];
+    for (let i = 0; i < theads.length; i++) {
```

Comment:
You can use `forEach` here instead of regular `for`

Reply:
This 'for loop' here is looping two different arrays, so I suppose it is difficult to use forEach here.